cg/nuget/README.md

```diff
@@ -0,0 +1,4 @@
+cgmanifest.json is an auto-generated file using the /generateCgManifestForNugets:<path> agrument while using bxl.
+Any changes to this file will be over written whenever we run the bxl command.
+If you are adding additional NuGet Packages to the project, they will automatically be added in the cgmanifest.json file whenever you build the project using the bxl command.
+If you need to add custom entries into a cgmanifest.json file, please create a new file with the name cgmanifest.json in a different folder anywhere within the project, since any changes to the cgmanifest.json in this folder wil always be over-written.
```

Comment:
I think the idea is that this is never necessary. This file is always auto-generated so if you need somethign updated you either need to update the source of truth (config.dsc) or change the source code of the generator